### DISCUSSION TITLE
Mirth Connect Administrator Launcher Source code

### DISCUSSION THREAD

**Discussion by mcswaip1** [nextgenhealthcare/connect#4981] [Feature/Enhancement Ideas]
Is the source code for the Mirth Admin Launcher in GH?
I was looking to add an Export/Import for the connections so that I can share my connections list with others in my organization.
Thanks

**Comment by jonbartels** [comment]
No the code is not available. The Launcher is free, but not open-source. 

Fortunately you can already share the connections file! Search in your installation directory for `connections.json` . That is where the connection data is stored. Just share and merge that file with your teammates.

**Reply by mcswaip1** [reply]
I can do that, but for the non-technical I wanted a simpler option.

**Reply by jonbartels** [reply]
Yep. An export/import option (maybe even with a smart merge or duplicate detection) would be really convenient. It is a good feature request.
